src/Components/Tasks/Tasks.tsx

```diff
@@ -10,17 +10,24 @@ import { Link } from "react-router-dom";
 import NoData from "../../Shared/NoData/NoData";
 // import AddTask from "../AddTask/AddTask";
 import noData from "./../../assets/images/no-data.png";
+import style from "./Tasks.module.css"
 // import Select from 'react-select';
-
+interface Itasks {
+  id: number;
+  status: string;
+  title: string
+}
 
 const Tasks: React.FC = () => {
   const [isLoading, setIsLoading] = useState(false);
-  const { baseUrl, requestHeaders }: any = useContext(AuthContext);
+  const { baseUrl, requestHeaders, userRole } = useContext(AuthContext);
   const { getToastValue }: any = useContext(ToastContext);
   const [userList, setUserList] = useState([])
   const [tasks, setTasks] = useState([]);
   const [taskDetails, setTaskDetails] = useState({});
-
+  const [todoTasks, setTodoTasks] = useState([])
+  const [progressTasks, setProgressTasks] = useState([])
```

Comment:
You can change the name to `InProgressTasks` to be similar to the one coming from the backend.